bluebottle/time_based/admin.py

```diff
@@ -23,10 +23,34 @@
 
 class BaseParticipantAdminInline(admin.TabularInline):
     model = Participant
-    readonly_fields = ('contributor_date', 'motivation', 'document', 'edit', 'created', 'transition_date', 'status')
+    readonly_fields = ('contributor_date', 'motivation', 'document', 'edit',
+                       'created', 'transition_date', 'status', 'disabled')
     raw_id_fields = ('user', 'document')
     extra = 0
 
+    def get_fields(self, request, obj=None):
+        if self.can_edit(obj):
+            return super().get_fields(request, obj)
+        else:
+            return ['disabled']
+
+    def get_template(self):
+        pass
+
+    def disabled(self, obj):
+        return format_html('<i>{}</i>', obj)
+    disabled.short_description = _('First complete and submit the activity before managing participants.')
```

Comment:
Granted, this is probably not the cleanest way to show a message like this, but it does the trick. Otherwise we have to override formset or somehow have conditional template.